modules/fastcv/test/test_bilateral.cpp

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2024 Qualcomm Innovation Center, Inc. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+*/
+
+#include "test_precomp.hpp"
+
+namespace opencv_test { namespace {
+
+typedef testing::TestWithParam<tuple<cv::Size,int,int>> fcv_bilateralFilterTest;
+
+TEST_P(fcv_bilateralFilterTest, accuracy)
+{	
+    cv::Size size  = get<0>(GetParam());
+	int d = get<1>(GetParam());
+    double sigmaColor = get<2>(GetParam());
+	double sigmaSpace = sigmaColor;
+	
+	RNG& rng = cv::theRNG();
+    Mat src(size, CV_8UC1);
+    cvtest::randUni(rng, src, Scalar::all(0), Scalar::all(256));
+
+    cv::Mat dst;
+
+	cv::fastcv::bilateralFilter(src, dst, d, sigmaColor, sigmaSpace);
+	
+    EXPECT_FALSE(dst.empty());
```

Comment:
Not enough checks
We should compare this image to some reference implementation (e.g. from OpenCV) or at least to source image with some fixed threshold, just to ensure that the result resembles smoothed image somehow.